flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/KafkaConsumerThread.java

```diff
@@ -68,17 +69,14 @@
 @Internal
 public class KafkaConsumerThread extends Thread {
 
-	/** Property that sets the boolean flag to enable rate-limiting. **/
-	private static final String USE_RATELIMITING = "kafka.consumer.ratelimit.enabled";
-
-	/** Property that sets the max bytes per second per consumer. **/
-	private static final String BYTES_PER_SECOND_MAX = "kafka.consumer.ratelimit.maxbytespersecond";
+	/** Consumer prefix for Ratelimiting. **/
+	private static final String CONSUMER_PREFIX = "kafka";
 
 	/** Default value for the rate-limiting flag. **/
 	private static final boolean DEFAULT_USE_RATELIMITING = false;
 
 	/** Default value for max bytes per second per consumer. **/
-	private static final long DEFAULT_BYTES_PER_SECOND_MAX = 1 * 1024 * 1024;
+	private static final long DEFAULT_BYTES_PER_SECOND_MAX = 1 * 1024 * 1024L;
```

Comment:
Get rid of this.  Handle in RateLimitingConfig.

Reply:
Done.